tests/foreman/virtwho/api/test_nutanix.py

```diff
@@ -0,0 +1,188 @@
+"""Test class for Virtwho Configure API
+
+:Requirement: Virt-whoConfigurePlugin
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Virt-whoConfigurePlugin
+
+:Assignee: yanpliu
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from nailgun import entities
+
+from robottelo.cli.host import Host
+from robottelo.cli.subscription import Subscription
+from robottelo.config import settings
+from robottelo.virtwho_utils import deploy_configure_by_command
+from robottelo.virtwho_utils import deploy_configure_by_script
+from robottelo.virtwho_utils import get_configure_command
+from robottelo.virtwho_utils import get_configure_file
+from robottelo.virtwho_utils import get_configure_option
+
+
+@pytest.fixture()
+def form_data(default_org, target_sat):
+    form = {
+        'name': gen_string('alpha'),
+        'debug': 1,
+        'interval': '60',
+        'hypervisor_id': 'hostname',
+        'hypervisor_type': settings.virtwho.ahv.hypervisor_type,
+        'hypervisor_server': settings.virtwho.ahv.hypervisor_server,
+        'organization_id': default_org.id,
+        'filtering_mode': 'none',
+        'satellite_url': target_sat.hostname,
+        'hypervisor_username': settings.virtwho.ahv.hypervisor_username,
+        'hypervisor_password': settings.virtwho.ahv.hypervisor_password,
+        'prism_flavor': settings.virtwho.ahv.prism_flavor,
+    }
+    return form
+
+
+@pytest.fixture()
+def virtwho_config(form_data):
+    return entities.VirtWhoConfig(**form_data).create()
```

Comment:
Consider using `target_sat.api.VirtWhoConfig` instead of importing entities. This is the preferred reference now - through the `target_sat` instance, providing more explicit link to the server config.

Reply:
If your form data_data becomes a class level scoped then even this can be class scoped.